client/src/index.html

```diff
@@ -2,10 +2,10 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Client</title>
+  <title>Lena & Lucas</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="icon" type="image/x-icon" href="/assets/favicon.png">
```

Comment:
Does it matter that its a png instead of an ico?

Reply:
Nah, ico is much more annoying to deal with though